contracts/container/contract.go

```diff
@@ -238,6 +239,42 @@ func Update(script []byte, manifest []byte, data any) {
 	runtime.Log("container contract updated")
 }
 
+// PutObject registers successful object PUT operation and notifies about
```

Comment:
what meaning do u put into `registers` here? In proposed implementation it verifies and notifies only

Reply:
> what meaning do u put

dont remember, maybe i meant how it really will be used: some meta service will wait for this notification. this is like the proof that an object exists. until this method is called it is not guaranteed